src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
```

Comment:
You shouldn't use tag names for styling (except `html` and `body`). However, in your case, since `body` is a unique element, it's acceptable to style it directly. Just be aware that for other tags, it's recommended to use classes for styling.